vendor/assets/stylesheets/ustyle/components/_hero.scss

```diff
@@ -24,6 +24,10 @@ $hero-text-color: $c-navy !default;
   @extend %container;
   max-width: 800px;
   padding: 2.5em 15px;
+
+  @include respond-to(desktop) {
```

Comment:
This is already in the grid? Can you confirm why this is being done?

Reply:
Had some inheritance issues previously. I have removed this now as the issue is no longer prevalent.